### PR TITLE
New hires checklist: how to build and run on personal iOS devices

### PR DESCRIPTION
**Pull request by horothesun**

### CODE REVIEW

**Review by karabatov — CHANGES_REQUESTED** [review]
This is essentially the same as in the [fastlane document](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Technical-Documents/FastlaneMatch.md#day-to-day-development), and I feel that this does not warn sufficiently about accidentally breaking `fastlane match` (which the linked document does).

On top of that, I don't think that it's good to maintain two places with a guide to do the same thing (run the app on your own device). I would prefer an edit to the fastlane document, perhaps with clearer steps, but keeping all the warnings intact, and linking to that from here.

Still, the intention to put this guide closer is a good one, because the fastlane document is a bit harder to discover.

**Comment by horothesun** [comment]
Thanks for the comment @karabatov ! I'll remove this duplicate paragraph putting a link to the fastlane document in the "What's next?" section 👍 
When I'll gather more knowledge about provisioning profiles/certificates I'll gladly open a dedicated PR for the fastlane document 😉

File: Cookbook/Technical-Documents/NewHiresCheckList.md [hunk 1]
```diff
@@ -92,6 +92,8 @@ shell script. Here's how to get the iOS project up and running:
 
 ## What's next?
 
+1. Refer to the [Fastlane Match](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Technical-Documents/FastlaneMatch.md) document for certificates and provisioning profiles related concerns.
```
Comment (karabatov) [inline Cookbook/Technical-Documents/NewHiresCheckList.md]:
```suggestion
1. Refer to the [Fastlane Match](FastlaneMatch.md#day-to-day-development) document if you want to run the app on your own iOS devices.
```

How about that? 
1. Relative link
1. Takes you directly to the relevant section
1. Clearer purpose of why you'd want to do it

**Review by karabatov — APPROVED** [review]

File: Cookbook/Technical-Documents/NewHiresCheckList.md (hunk 1 above)
Comment (horothesun) [reply Cookbook/Technical-Documents/NewHiresCheckList.md]: Sounds good! 👍

**Review by MichalTKwiecien — APPROVED** [review]

**Pull request merged**